### ISSUE TITLE
Ability to dismiss a triggering category/mood/strategy in the stats module

### ISSUE THREAD

**Issue by julianguyen** [ifmeorg/ifme#259]
# Description

The stats aggregate the the the top (max 3, min 1) categories, moods, and strategies. However, through personal use of the app, I've realized that seeing certain buttons can be triggering e.g. "something triggering". It would be neat to have something to dismiss these from view in the stats module and go back and allow them to be in the app again.

![screen shot 2016-08-27 at 12 04 49 pm](https://cloud.githubusercontent.com/assets/3010728/18029510/a013151c-6c4e-11e6-93d3-2c86db8fd690.png)

---

Please assign yourself (via the Assignees dropdown), if you do want to work on this issue. Can't find yourself? You need to [join our organization](https://github.com/ifmeorg/ifme/wiki/Join-Our-Slack).

Check out our [Picking Up Issues](https://github.com/ifmeorg/ifme/wiki/Picking-Up-Issues) guide if you haven't already!

**Comment by julianguyen** [comment]
@rashmiagar Let me know if you want to tackle this! :)

**Comment by rashmiagar** [comment]
@julianguyen Ya, definitely :)

**Comment by julianguyen** [comment]
Wonderful @rashmiagar :) Let's sync on the design for this sometime. Right now when you hover over the text, there's a pop up div showing which of your allies can view category/mood/strategy. We'll have to figure out a state that allows some kind of "X" button to show up to dismiss the button. And then from a data standpoint, have that flag saved.

**Comment by julianguyen** [comment]
Hey @rashmiagar. Any updates?

**Comment by su6a12** [comment]
Hi Julia, I'd like to help out with this one if it's no longer assigned.

**Comment by julianguyen** [comment]
Awesome I'll assign it to you @su6a12! As always, feel free to ask questions/chat more about it on Slack!

**Comment by julianguyen** [comment]
How is this going @su6a12?

**Comment by elynnyap** [comment]
Hey Julia! @aftaberski and I just started on this tonight and would like to keep working on it :)

**Comment by fumba** [comment]
@julianguyen  I am getting myself familiar with this app and wasn't sure where this feature would be implemented in the most current version of the app. I logged in with test user - `test2@example.com` and this is the only thing I could see. 

<img width="1669" alt="screen shot 2019-03-05 at 9 10 56 pm" src="https://user-images.githubusercontent.com/7467648/53855547-53c60080-3f8b-11e9-92d8-0fea1c99ba83.png">

Can you please help guide me through this proposed feature? Do I need to log in as a different kind of user?

**Comment by jack-rl-ovell** [comment]
Hi @julianguyen is there still help needed on this project? Would love to help!

**Comment by julianguyen** [comment]
I believe @Iphytech is working on this! :D You could help out if it's needed!

Otherwise, let me know if you need help finding another issue to work on!

**Comment by alcoelho** [comment]
Hey, I want to help with this feature, but I'm not sure if I'm understanding what has to be done. Can someone explain it to me? 
When I open the Categories section, I can't see the mentioned bar.

**Comment by julianguyen** [comment]
@alcoelho We would love someone to work on this! So whenever you tag Moments with Categories, Moods, and Strategies and Strategies with Categories and Moods we display data on Categories, Moods, and Strategies that have been most recently used. Sometimes whatever these things are can be triggering due to trauma. We want the ability to hide these values from being displayed in this data section.

So basically we want to add `visible: boolean` to the data model and add the ability to toggle it so that it's visible/not visible in the data section.

**Comment by alcoelho** [comment]
I see, that's a good thing to add. I want to work on it :)

**Comment by Buzunda** [comment]
Hey @alcoelho do you need any help with this?

**Comment by Buzunda** [comment]
Is this what you had in mind for the toggle part? 

![Screen Shot 2020-01-27 at 22 01 43](https://user-images.githubusercontent.com/4948776/73276034-cdc82f00-41c6-11ea-8682-ea69a5a50605.png)

**Comment by julianguyen** [comment]
@Buzunda I think that works! Great suggestion :)

**Comment by Buzunda** [comment]
ok sounds good. 
How do I handle the copy text for all the other languages? Can I trust google translate?

**Comment by julianguyen** [comment]
@Buzunda Yeah you'll have to use Google Translate for now since the folks who did our translations aren't always available.
